packages/web/src/pages/saved-page/SavedPageProvider.tsx

```diff
@@ -101,6 +101,7 @@ class SavedPage extends PureComponent<SavedPageProps, SavedPageState> {
   }, 300)
 
   handleFetchMoreSavedTracks = (offset: number, limit: number) => {
+    if (this.props.endFetchStatus) return
```

Comment:
"If endFetchStatus" seems kind of vague, maybe would be good to name it something like `hasReachedEnd` or `hasReachedEndOfFetchableItems` if we wanna be really verbose. Or we could add a comment here to explain it a bit